src/Symfony/Component/Security/Core/Authorization/AuthorizationCheckerInterface.php

```diff
@@ -21,10 +21,10 @@ interface AuthorizationCheckerInterface
     /**
      * Checks if the attributes are granted against the current authentication token and optionally supplied subject.
      *
-     * @param mixed $attributes
-     * @param mixed $subject
+     * @param string $attribute
```

Comment:
That's wrong, `$attribute` can be anything and not just strings, we actually use non-string attributes in the expression voter that expect an `Expression` object.

Reply:
@wouterj the feature mentioned by @jvasseur looks uncommon, but valid nonetheless, right? If you agree, we should replace `string` by `mixed` in the PHPdoc.